wger/exercises/views/muscles.py

```diff
@@ -36,10 +36,14 @@ class MuscleListView(ListView):
     Overview of all muscles and their exercises
     '''
     model = Muscle
-    queryset = Muscle.objects.all().order_by('-is_front', 'name'),
     context_object_name = 'muscle_list'
     template_name = 'muscles/overview.html'
 
+    def get_queryset(self):
```

Comment:
LGTM, wanted to know though why the if statement is required, running on my mac I have noted it cannot run without it, so its important, may be you share the knowledge...

Reply:
The if statement is because when you request for multiple objects and you are using the get_queryset method, for it to iterate through them the request should have a comma at the end, but when I tried requesting for one object it was giving an error with the comma, so I added the if statement to alter the statement when requesting for one object and for multiple object